Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -128,6 +129,7 @@ public struct BuildDescription {
                 return PluginTargetBuildDescription(
                     target: target,
                     toolsVersion: package.manifest.toolsVersion,
+                    toolchain: buildPlan.toolsBuildParameters.toolchain as? UserToolchain,
```

Comment:
I very much dislike this, but `ManifestLoader` has a bunch of uses of members in `UserToolchain` that are not in `Toolchain`. Should we add requirements for them all?

Reply:
If they make sense for Toolchain protocol, why not?

Reply:
There is a FIXME right on top of UserToolchain, might be the time to make it right...